playwright/integration/auth.spec.ts

```diff
@@ -14,7 +14,7 @@ test.beforeEach(async ({ context }) => {
 test.afterEach(() => server.resetHandlers());
 test.afterAll(() => server.close());
 
-test.describe("Cookie authentication", () => {
+test.skip("Cookie authentication", () => {
```

Comment:
does this test still not pass? should we skip?

Reply:
The tests didn't pass due to a double redirect for the login.nypl.org page. I updated the tests to check for the final destination.